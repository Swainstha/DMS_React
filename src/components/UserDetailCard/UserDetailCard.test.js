import testSnapFunction from '../../util/testSnapFunction';
import UserDetailCard from './index'
import React from 'react';



testSnapFunction("<itemSecond/>", "Snapshot test for ProductGridItem",<UserDetailCard /> )