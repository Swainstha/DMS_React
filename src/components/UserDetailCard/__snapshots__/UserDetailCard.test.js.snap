// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<itemSecond/> Snapshot test for ProductGridItem 1`] = `ShallowWrapper {}`;
